src/algorithms/standard/vectorrealtotensor.cpp

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "vectorrealtotensor.h"
+
+using namespace std;
+
+namespace essentia {
+namespace streaming {
+
+const char* VectorRealToTensor::name = "VectorRealToTensor";
+const char* VectorRealToTensor::category = "Standard";
+const char* VectorRealToTensor::description = DOC("This algorithm takes a stream of frames "
+"and outputs them as a single tensor everytime the output shape is reached. ")
+"If the batch dimension (0) is set to -1 the data will be accumulated and returned "
```

Comment:
Explain terminology better: what is the difference between batch, patch, and frames.